fft/unit_test/Test_Transform.cpp

```diff
@@ -295,28 +299,26 @@ void test_fft1_1dfft_1dview() {
   ComplexView1DType out_b("out_b", len), out_o("out_o", len),
       out_f("out_f", len);
 
+  execution_space exec;
   const Kokkos::complex<T> z(1.0, 1.0);
-  Kokkos::Random_XorShift64_Pool<> random_pool(12345);
-  Kokkos::fill_random(x, random_pool, z);
-
-  Kokkos::fence();
+  Kokkos::Random_XorShift64_Pool<execution_space> random_pool(12345);
+  Kokkos::fill_random(exec, x, random_pool, z);
+  exec.fence();
 
-  KokkosFFT::fft(execution_space(), x,
-                 out);  // default: KokkosFFT::Normalization::backward
-  KokkosFFT::fft(execution_space(), x, out_b,
-                 KokkosFFT::Normalization::backward);
-  KokkosFFT::fft(execution_space(), x, out_o, KokkosFFT::Normalization::ortho);
-  KokkosFFT::fft(execution_space(), x, out_f,
-                 KokkosFFT::Normalization::forward);
+  KokkosFFT::fft(exec, x, out);  // default: KokkosFFT::Normalization::backward
+  KokkosFFT::fft(exec, x, out_b, KokkosFFT::Normalization::backward);
+  KokkosFFT::fft(exec, x, out_o, KokkosFFT::Normalization::ortho);
+  KokkosFFT::fft(exec, x, out_f, KokkosFFT::Normalization::forward);
 
-  fft1(execution_space(), x, ref);
-  multiply(execution_space(), out_o, sqrt(static_cast<T>(len)));
-  multiply(execution_space(), out_f, static_cast<T>(len));
+  fft1(exec, x, ref);
+  multiply(exec, out_o, sqrt(static_cast<T>(len)));
```

Comment:
Not related to this PR but I see an unqualified call to `sqrt`, aren't we calling the C version in this case ?

Reply:
Oh, that is true. 
We should use `std::sqrt` or `Kokkos::sqrt`

Reply:
We can `grep` later and change this in a new PR.

Reply:
I made it in this PR. After CI passes, I will merge this

Reply:
ok